dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/FirstOrderLimiter.mo

```diff
@@ -0,0 +1,65 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2021, RTE (http://www.rte-france.com) and UPC/Citcea (https://www.citcea.upc.edu/)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block FirstOrderLimiter "First order filter with non-windup limiter"
```

Comment:
This block is already in Blocks/NonLinear (as it is a limiter) so we can remove this new one and use the one in NonLinear

Reply:
Done. Deleting the added block and using the existing one. Thanks for noticing it!